src/components/Column/Column.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { css } from "emotion";
+
+type Props = { children: JSX.Element | JSX.Element[] };
+
+export const Column = ({ children }: Props) => (
+  <div
+    className={css`
+      display: flex;
```

Comment:
by having inlines styles we generate a new key for every use. In order for them to share the same key and reduce the css bundle size we could maybe define a const outside the component?

Reply:
> by having inlines styles we generate a new key for every use

I never realised this. Is there a link to the docs about this? We use the inline pattern a lot in the codebase, I wonder if there's a solution higher up that we could look at rather than having to refactor everything?

Reply:
Yeah interesting!

Reply:
I ran a couple tests to see how keys are generated. 

They keys seem generated by component scope, so you are right @oliverlloyd  there is actually no difference. 

I assumed css command worked like a function and was concerned that calling it inline would be like calling `css(...)` inline causing a recalculation every time.

Some findings I made, if you define
```
const someStyles = css`
  width: 30px;
  color: blue;
`

css`
  width: 30px;
  color: blue;
`
 ```
both will generate different CSS keys, so you either need to stick to one or the other in the same Component.

Reply:
Awesome investigation! Shame there's not a perf win to be had but it's useful to learn about the inline vs. const difference inside a component 👍